javascript/js-loops_stars/js/index.js

```diff
@@ -2,13 +2,26 @@ console.clear();
 
 const starContainer = document.querySelector('[data-js="star-container"]');
 
-function renderStars() {
+function renderStars(filledStars) {
   // reset the star container before rerendering stars
   starContainer.innerHTML = "";
 
   //--v-- your code here --v--
 
-  //--^-- your code here --^--
+  for (let i = 1; i <= 5; i++) {
+    const image = document.createElement("img");
+    if (filledStars >= i) {
+      image.src = "../assets/star-filled.svg";
```

Comment:
Path here should just be 'assets/star-filled.svg' and same for the empty stars.

Reply:
that did not work for me, i had to go one folder up. did it work for you??